flow/hash.go

```diff
@@ -38,6 +38,29 @@ func (fl *ICMPLayer) Hash(hasher hash.Hash) {
 	hasher.Write(value32)
 }
 
+func (tl *TransportLayer) Hash(hasher hash.Hash) {
```

Comment:
As you change the hash function you have to change the TrackingID for all the unit test.